app/models/concerns/has_modification_section.rb

```diff
@@ -8,10 +8,6 @@ module HasModificationSection
   end
 
   def first_modification
-    modifications.first
-  end
-
-  def section
-    @section ||= first_modification.section if modifications?
+    modifications&.first
```

Comment:
can `modifications` be `nil`?

Reply:
No, it shouldn't.
```
2.6.3 :003 > Amendments::StandardAmendment.new.modifications
 => #<ActiveRecord::Associations::CollectionProxy []>
```
Fixed here: https://github.com/podemos-info/national_budgets/commit/96b5681b9e4ddc61e0f5f1d907da195abdbf5dda